src/rgw/rgw_aio.cc

```diff
@@ -58,7 +58,7 @@ Aio::OpFunc aio_abstract(librados::IoCtx ctx, Op&& op, jspan_context* trace_ctx
         (void)trace_ctx; // suppress unused trace_ctx warning. until we will support the read op trace
```

Comment:
Why to revert the merge commit? Wouldn't be enough to simply:

```
$ git revert 9f160e4
[detached HEAD c3b26d99f2c] Revert "rgw/aio: fix recursion from tracing changes"
 1 file changed, 2 insertions(+), 2 deletions(-)
$ git revert 5d4c5e6
Auto-merging src/rgw/rgw_op.cc
Auto-merging src/osd/OSD.cc
Auto-merging src/msg/Message.h
Auto-merging src/msg/Message.cc
Auto-merging src/messages/MOSDOp.h
Auto-merging src/common/tracer.h
[detached HEAD 147f743d2c0] Revert "tracer/osd/librados/build/rgw: rgw and osd end2end tracing using opentelemetry"
 33 files changed, 139 insertions(+), 233 deletions(-)
```
?

Reverting regular, individual commits is straightforward and results in pretty readable history:

```
$ git log --oneline -n3
147f743d2c0 (HEAD) Revert "tracer/osd/librados/build/rgw: rgw and osd end2end tracing using opentelemetry"
c3b26d99f2c Revert "rgw/aio: fix recursion from tracing changes"
92ba1c08c83 (upstream/main) Merge pull request #50324 from pkalever/scrub_incomplete
```

Reverting a merge commit is, well, far more complex underneath and can be quirky: https://mirrors.edge.kernel.org/pub/software/scm/git/docs/howto/revert-a-faulty-merge.txt

Reply:
agree. will revert the actual commit

Reply:
pushed a version reverting the actual commit and not the merge